src/ResourceManager/Search/Search.sln

```diff
@@ -0,0 +1,73 @@
+﻿
```

Comment:
You need to add these projects to get the packaging job to pass: 
https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Aks/Aks.sln#L28
https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Aks/Aks.sln#L24